packages/http-client-csharp/emitter/src/type/input-type.ts

```diff
@@ -122,7 +122,7 @@ export interface InputModelType extends InputTypeBase {
 }
 
 export interface InputModelProperty extends InputTypeBase {
-  kind: "property";
+  kind: InputModelPropertyKind;
```

Comment:
could we here just simply reference the union definition from TCGC instead of define our own?
It could be something like
```suggestion
  kind: SdkPropertyType["kind"];
```

Reply:
sure.